scripts/presubmit-tests.sh

```diff
@@ -88,6 +88,10 @@ function main() {
         RUN_INTEGRATION_TESTS=1
         shift
         ;;
+      --upgrade-tests)
```

Comment:
You can revert this file, https://github.com/knative/test-infra/pull/256 implements running arbitrary tests on presubmit.